src/NUnitFramework/framework/Constraints/PropertyConstraintResult.cs

```diff
@@ -0,0 +1,40 @@
+namespace NUnit.Framework.Constraints
+{
+    /// <summary>
+    /// Contain the result of matching a <see cref="PropertyConstraint"/> against an actual value.
```

Comment:
For a separate PR since it needs to be done across all our constraint results:
ℹ Should be `Contains`, as in, “PropertyConstraintResult contains the result…”
Further, does PropertyConstraintResult *contain* the result, or *is* it the result?